tls/s2n_psk.h

```diff
@@ -80,6 +80,12 @@ int s2n_psk_free(struct s2n_psk *psk);
 S2N_RESULT s2n_psk_parameters_init(struct s2n_psk_parameters *params);
 S2N_CLEANUP_RESULT s2n_psk_parameters_wipe(struct s2n_psk_parameters *params);
 
+struct s2n_psk_identity_list {
+    struct s2n_stuffer *identities;
+};
+S2N_RESULT s2n_psk_identity_list_get(struct s2n_psk_identity_list *list, uint16_t index,
```

Comment:
Nit: Add a space between struct and function definition

Reply:
I kind of like it this way. It shows that the methods operate on that structure, like how we don't put newlines between closely related function definitions.